adagrams/game.py

```diff
@@ -1,13 +1,100 @@
 from random import randint
 
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
```

Comment:
Nice work on using the python convention of having your variable name in uppercase (constant variable). This signals to other developers that this variable should not be modified or reassigned. Also, I would suggest maybe condensing the amount of lines that you have for dictionary just for readability.
```suggestion
LETTER_POOL = {
	    "A":9,"B":2,"C":2,"D":4,
	    "E":12,"F":2,"G":3,"H":2,
	    "I":9,"J":1,"K":1,"L":4,"M":2,
	    "N":6,"O":8,"P":2,"Q":1,
	    "R":6,"S":4,"T":6,"U":4,
	    "V":2,"W":2,"X":1,
	    "Y":2,"Z":1
	}
```